pom.xml

```diff
@@ -28,11 +74,11 @@
         <profile.swagger />
 
         <!-- Dependency versions -->
-        <jhipster-dependencies.version>0.1.4</jhipster-dependencies.version>
+        <jhipster-dependencies.version>2.0.0-20180219.202458-25</jhipster-dependencies.version>
```

Comment:
We have a newer version with Spring Boot RC2 you should try:

        <jhipster-dependencies.version>2.0.0-20180224.150007-31</jhipster-dependencies.version>
        <!-- The spring-boot version should match the one managed by
        https://mvnrepository.com/artifact/io.github.jhipster/jhipster-dependencies/${jhipster-dependencies.version} -->
        <spring-boot.version>2.0.0.RC2</spring-boot.version>
        <spring-cloud-netflix.version>2.0.0.BUILD-SNAPSHOT</spring-cloud-netflix.version>
        <spring.cloud.version>Finchley.BUILD-SNAPSHOT</spring.cloud.version>